Tests/MapboxNavigationTests/LaneTests.swift

```diff
@@ -31,7 +31,7 @@ class LaneTests: FBSnapshotTestCase {
                 let groupView = UIStackView(orientation: .vertical, autoLayout: true)
                 groupView.alignment = .center
                 
-                let laneView = LaneView(indications: lane.indications, isUsable: true)
+                let laneView = LaneView(indications: lane.indications, isUsable: true, direction: ManeuverDirection(rawValue: lane.indications.description))
```

Comment:
It’s incorrect to create a `ManeuverDirection` from a `LaneIndications.description`. The description could be a comma-separated list.

Reply:
Added specific maneuver direction to TestableLane struct.